libcxx/test/std/utilities/tuple/tuple.tuple/tuple.rel/size_incompatible_comparison.verify.cpp

```diff
@@ -21,9 +21,15 @@
 
 #include <tuple>
 
+#include "test_macros.h"
+
+#if TEST_STD_VER >= 26
```

Comment:
Can we just make this UNSUPPORTED: std-at-least-cxx26? Also, please add a comment why this is disabled since C++26.